Library/Formula/xsd.rb

```diff
@@ -0,0 +1,98 @@
+class Xsd < Formula
+  desc "XML Data Binding for C++"
+  homepage "http://www.codesynthesis.com/products/xsd/"
+  url "http://www.codesynthesis.com/download/xsd/4.0/xsd-4.0.0+dep.tar.bz2"
+  version "4.0.0"
+  sha256 "eca52a9c8f52cdbe2ae4e364e4a909503493a0d51ea388fc6c9734565a859817"
+
+  depends_on "pkg-config" => :build
+  depends_on "xerces-c"
+
+  # As of version 4.0.0, Clang fails to compile if the <iostream> header is not
+  # explicitly included. The developers are aware of this problem, see:
+  # http://www.codesynthesis.com/pipermail/xsd-users/2015-February/004522.html
+  patch <<-EOS.undent
```

Comment:
Inline patches need to go at the end under an `__END__` header beneath the main formula. It keeps the main formula body clean and easily readable. Then just:

```ruby
# Patch 1 does xyz
# Reported Patch 1 here: 
# Patch 2 does xyz
# Reported Patch 2 here:
patch :DATA
```